jira7/jira_endpoint.bal

```diff
@@ -278,28 +274,32 @@ public remote function Client.getAllDetailsFromProjectSummary(ProjectSummary pro
     if(projectOut is Project) {
         return projectOut;
     } else {
-        return errorToJiraConnectorError(projectOut);
+        error err = error(CONVERSION_ERROR_CODE, { cause: projectOut,
+            message: "Error occurred while doing json conversion." });
```

Comment:
```suggestion
            message: "Error occurred on JSON to record conversion." });
```

Reply:
Do we need to wrap this error anyway? Can't we just say `convertJsonToProject(jsonResponseOut);`